lib/store.js

```diff
@@ -9,6 +9,15 @@ function isFunc(object) {
   return (typeof object === 'function');
 }
 
+function devTools () {
+  if (typeof window === 'object'
+      && typeof window.devToolsExtension !== 'undefined') {
```

Comment:
Hmmm... I guess you're right. Worst that can happen is someone
hooks on our store with their DevTools, it's not like they access the
store itself so it should be fine.